modules/tensor_mechanics/tests/Rayleigh_damping/Newmark_time_integration/Rayleigh_Newmark.i

```diff
@@ -0,0 +1,325 @@
+  [./TensorMechanics]
+    displacements = 'disp_x disp_y disp_z'
+  [../]
+  [./DynamicTensorMechanics]
+    displacements = 'disp_x disp_y disp_z'
+    zeta = 0.1
+  [../]
+  [./inertia_x]
+    type = InertialForce
+    variable = disp_x
+    velocity = vel_x
+    acceleration = accel_x
+    beta = 0.25
+    gamma = 0.5
+    eta=0.1
+  [../]
+  [./inertia_y]
+    type = InertialForce
+    variable = disp_y
+    velocity = vel_y
+    acceleration = accel_y
+    beta = 0.25
+    gamma = 0.5
+    eta=0.1
+  [../]
+  [./inertia_z]
+    type = InertialForce
+    variable = disp_z
+    velocity = vel_z
+    acceleration = accel_z
+    beta = 0.25
+    gamma = 0.5
+    eta = 0.1
+  [../]
+
+[]
+
+[AuxKernels]
+  [./accel_x]
+    type = NewmarkAccelAux
+    variable = accel_x
+    displacement = disp_x
+    velocity = vel_x
+    beta = 0.25
+    execute_on = timestep_end
+  [../]
+  [./vel_x]
+    type = NewmarkVelAux
+    variable = vel_x
+    acceleration = accel_x
+    gamma = 0.5
+    execute_on = timestep_end
+  [../]
+  [./accel_y]
+    type = NewmarkAccelAux
+    variable = accel_y
+    displacement = disp_y
+    velocity = vel_y
+    beta = 0.25
+    execute_on = timestep_end
+  [../]
+  [./vel_y]
+    type = NewmarkVelAux
+    variable = vel_y
+    acceleration = accel_y
+    gamma = 0.5
+    execute_on = timestep_end
+  [../]
+  [./accel_z]
+    type = NewmarkAccelAux
+    variable = accel_z
+    displacement = disp_z
+    velocity = vel_z
+    beta = 0.25
+    execute_on = timestep_end
+  [../]
+  [./vel_z]
+    type = NewmarkVelAux
+    variable = vel_z
+    acceleration = accel_z
+    gamma = 0.5
+    execute_on = timestep_end
+  [../]
+  [./stress_yy]
+     type = RankTwoAux
+     rank_two_tensor = stress
+     variable = stress_yy
+     index_i = 0
+     index_j = 1
+  [../]
+  [./strain_yy]
+     type = RankTwoAux
+     rank_two_tensor = total_strain
+     variable = strain_yy
+     index_i = 0
+     index_j = 1
+  [../]
+
+[]
+
+
+[BCs]
+  [./top_y]
+    type = DirichletBC
+    variable = disp_y
+    boundary = top
+    value=0.0
+  [../]
+  [./top_x]
+   type = DirichletBC
+    variable = disp_x
+    boundary = top
+    value=0.0
+  [../]
+  [./top_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = top
+    value=0.0
+  [../]
+  [./bottom_x]
+    type = DirichletBC
+    variable = disp_x
+    boundary = bottom
+    value=0.0
+  [../]
+  [./bottom_z]
+    type = DirichletBC
+    variable = disp_z
+    boundary = bottom
+    value=0.0
+  [../]
+  [./Pressure]
+    [./Side1]
+    boundary = bottom
+    function = pressure
+    disp_x = disp_x
+    disp_y = disp_y
+    disp_z = disp_z
+    factor = 1
+    [../]
+  [../]
+[]
+
+[Materials]
+
+  [./Elasticity_tensor]
+     type = ComputeElasticityTensor
+     block = 0
+     fill_method = symmetric_isotropic
+     C_ijkl = '210e9 0'
+  [../]
+
+  [./strain]
+        type = ComputeSmallStrain
+        block = 0
+        displacements = 'disp_x disp_y disp_z'
+  [../]
```

Comment:
Regarding disp(3) vs disp(LIBMESH_DIM), I used disp(3) as the
StressDivergenceTensors.C (which already existed) uses disp(3).

If the number of displacements provided is not equal to the mesh dimension,
then both StressDivergenceTensor.C and DynamicStressDivergenceTensors.C
kernels give an error message.

I will format the test files and resubmit.

Thanks,
Swetha

On Wed, Sep 9, 2015 at 1:26 PM, Cody Permann <notifications@github.com>
wrote:

> In
> modules/tensor_mechanics/tests/Rayleigh_damping/Newmark_time_integration/Rayleigh_Newmark.i
> <https://github.com/idaholab/moose/pull/5671#discussion_r39085328>:
>
> > +[]
> > +
> > +[Materials]
> > +
> > +  [./Elasticity_tensor]
> > +     type = ComputeElasticityTensor
> > +     block = 0
> > +     fill_method = symmetric_isotropic
> > +     C_ijkl = '210e9 0'
> > +  [../]
> > +
> > +  [./strain]
> > +        type = ComputeSmallStrain
> > +        block = 0
> > +        displacements = 'disp_x disp_y disp_z'
> > +  [../]
>
> You should use consistent indentation in your input files too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5671/files#r39085328>.
>